lib/test/test-panel.js

```diff
@@ -14,6 +19,36 @@ export default class TestPanel extends EtchComponent {
     if (this.props.model) {
       this.props.model.view = this
     }
+
+    this.locationRegex = /([\w-/.\\:]*.go:\d+)/g
```

Comment:
going to leave this as is - even with a column we'll still link to the correct line